### DISCUSSION TITLE
Full stack framework like Drayman

### DISCUSSION THREAD

**Discussion by ansarizafar** [nanojsx/nano#93] [Ideas]
It would be great If we can develop Drayman http://www.drayman.io/ like full stack framework with nanojsx.

**Comment by yandeu** [comment]
What exactly do you mean by that? What should be added to make a drayman like framework?

**Comment by ansarizafar** [comment]
Drayman is a full stack framework we don't need API endpoints for data/business logic. Its a great time saver but If you don't like the idea then at least we should have something like React server components.

**Reply by jrson83** [reply]
I also don't understand exactly what you mean. Do you mean [Full stack component](http://www.drayman.io/docs/introduction/server-and-client-combined)? If so, this is build in in Nano already, SSR & Isomorphic Router?

**Comment by ansarizafar** [comment]
Drayman allows using server-side and client-side code together. This allows you to easily combine server-side libraries, file-system queries, database connections, managing user-sensitive data and creating UI inside single script.
Is this possible with nano? How can we write the same example with nano which is on http://www.drayman.io/ home page.

**Reply by yandeu** [reply]
Nanojsx is nearly 100% Isomorphic. So you can use amy code on the client and server.

I don't really see the purpos of drayman. What does it do? It seems like it is only remdering some jsx on the server and is embedded as a WebComponent on the client?
